python/cuml/test/dask/conftest.py

```diff
@@ -1,17 +1,22 @@
 import pytest
 
+from dask_cuda import initialize
 from dask_cuda import LocalCUDACluster
 
+enable_tcp_over_ucx = True
+
 
 @pytest.fixture(scope="module")
-def ucx_cluster():
-    cluster = LocalCUDACluster(protocol="ucx", threads_per_worker=1)
+def cluster():
+    cluster = LocalCUDACluster()
     yield cluster
     cluster.close()
 
 
 @pytest.fixture(scope="module")
-def cluster():
-    cluster = LocalCUDACluster(threads_per_worker=1)
+def ucx_cluster():
+    initialize.initialize(create_cuda_context=True,
+                          enable_tcp_over_ucx=enable_tcp_over_ucx)
+    cluster = LocalCUDACluster(protocol="ucx")
```

Comment:
You should add `enable_tcp_over_ucx=enable_tcp_over_ucx` here as well. I think we will remove that requirement later, but for now we're extra verbose so that we can control all flags correctly.

Reply:
@pentschev, I had removed that option from `LocalCUDACluster` because I'm getting the following error on `branch-0.11` of `dask-cuda`

```
Process Dask Worker process (from Nanny):
Traceback (most recent call last):
  File "/share/conda/cuml_11_3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/share/conda/cuml_11_3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/share/conda/cuml_11_3/lib/python3.7/site-packages/distributed/process.py", line 191, in _run
    target(*args, **kwargs)
  File "/share/conda/cuml_11_3/lib/python3.7/site-packages/distributed/nanny.py", line 674, in _run
    worker = Worker(**worker_kwargs)
  File "/share/conda/cuml_11_3/lib/python3.7/site-packages/distributed/worker.py", line 634, in __init__
    **kwargs
TypeError: __init__() got an unexpected keyword argument 'enable_tcp_over_ucx'

```

@dantegd mentioned he was seeing this error as well.

Reply:
Just noticed your latest PR was merged so I've updated the code to reflect those changes.

Reply:
Yes, before that PR you'd need to use `DGX`, so hopefully this is now cleaner. Sorry for the last minute changes. :)